components/automate-cli/pkg/client/apiclient/apiclient.go

```diff
@@ -25,7 +24,6 @@ type client struct {
 	authzClient        authz.AuthorizationClient
 	teamsClient        teams.TeamsClient
 	teamsV2Client      v2.TeamsClient
-	tokensClient       tokens.TokensMgmtClient
 	tokensV2Client     v2.TokensClient
```

Comment:
let's drop `v2` where we can (including the `v2` import name up on L17)

```suggestion
	tokensClient    iam.TokensClient
```

Reply:
Oh yeah I was gonna do that on a final pass. Thanks!